drools-ha/ha-core-infra/src/test/java/org/kie/hacep/PodAsReplicaTest.java

```diff
@@ -34,6 +35,7 @@
 
     private Logger logger = LoggerFactory.getLogger(PodAsReplicaTest.class);
 
+    @Ignore //see https://github.com/kiegroup/droolsjbpm-integration/pull/2156
```

Comment:
Can we have here a link to Jira about enabling and stabilizing the test, please?

Reply:
The test locally is always green, it start a kafka embedded instance, running on other test under heavy load could spent more that 30 sec and a timeout exception is throw, can I see the stack trace of the error ?